lutris/gui/lutriswindow.py

```diff
@@ -996,28 +996,39 @@ def on_sidebar_changed(self, widget):
 
     @watch_errors()
     def on_game_selection_changed(self, view, selection):
+        game_ids = []
+        games = []
         if not selection:
             GLib.idle_add(self.update_revealer)
             return False
-        game_id = view.get_model().get_value(selection, COL_ID)
-        if not game_id:
+        if type(selection) != list:
+            game_id = view.get_model().get_value(selection, COL_ID)
+            game_ids.append(game_id)
+        else:
+            for path in selection:
+                iterator = view.get_model().get_iter(path)
+                game_id = view.get_model().get_value(iterator, COL_ID)
+                game_ids.append(game_id)
+        if not game_ids:
             GLib.idle_add(self.update_revealer)
             return False
-        if self.service:
-            game = ServiceGameCollection.get_game(self.service.id, game_id)
-        else:
-            game = games_db.get_game_by_field(int(game_id), "id")
-        if not game:
-            game = {
+        for game_id in game_ids:
+            if self.service:
+                game = ServiceGameCollection.get_game(self.service.id, game_id)
+            else:
+                game = games_db.get_game_by_field(int(game_id), "id")
+            games.append(game)
+        if not games:
+            games = [{
```

Comment:
This 'if not games' bit is wrong; this needs to be inside the look, to generate a placeholder for *each* game that is not found. As written I think it never runs, as you will wind up with a list-of-one-element, containing only None, if you select a game that is not found in the database.

Reply:
It's supposed to generate placeholder if games list is empty after the loop. It's how it is done right now in the master branch too.

Reply:
No, the master branch has no list here; it does the placeholder if the (single) game is not found.

I think that logic should be per game, rather than meaning something like 'if no games are selected, make a placeholder somehow'.

Reply:
Ok. Will make it work like that then

Reply:
Thank you.